modules/apps/notification/notification-service/service.xml

```diff
@@ -47,6 +47,31 @@
 			<finder-column comparator="&lt;" name="sentDate" />
 		</finder>
 	</entity>
+	<entity local-service="true" name="NotificationQueueEntryAttachment" remote-service="false" table="NQueueEntryAttachment" uuid="false">
+
+		<!-- PK fields -->
+
+		<column db-name="NQueueEntryAttachmentId" name="notificationQueueEntryAttachmentId" primary="true" type="long" />
+
+		<!-- Audit fields -->
+
+		<column name="companyId" type="long" />
```

Comment:
I don't think we need. I will remove them.

Reply:
Done! I
 kept the `companyId` because of the SF: `Entity 'NotificationTemplateAttachment' should have a column named 'companyId', See LPS-107076:`, and I also kept the `userId`.

Reply:
Hey, @carolmariaabb. I think we can remove the userId too, because we have plenty examples of relationship tables that don't have it (DDMStructureLink, DEDataDefinitionFieldLink, DDMStorageLink etc). Do you see any usage for it?

Reply:
Not. Actually, I wasn't sure if it was okay to remove. Thanks for showing me these examples. I will remove it now.